lib/ex_unit/lib/ex_unit/doc_test.ex

```diff
@@ -283,37 +289,47 @@ defmodule ExUnit.DocTest do
   defp gen_code_for_tests(tests, whole_expr, stack) do
     quote do
       stack = unquote(stack)
+
       try do
         # Put all tests into one context
-        unquote_splicing(tests)
+        (unquote_splicing(tests))
       rescue
         e in [ExUnit.AssertionError] ->
           reraise e, stack
 
         error ->
-          original_stack = System.stacktrace
+          original_stack = System.stacktrace()
+
           reraise ExUnit.AssertionError,
```

Comment:
Can you plaese rewrite this one? You should compute the message on its own line, something like:

```
reraise ExUnit.AssertionError,
        [message: message, expr: unquote(String.trim(whole_expr))],
        original_stack
```